lib/LLVMPasses/ARCEntryPointBuilder.h

```diff
@@ -252,11 +252,16 @@ class ARCEntryPointBuilder {
     
     auto *ObjectPtrTy = getObjectPtrTy();
     auto &M = getModule();
-    auto AttrList = AttributeList::get(M.getContext(), 1, Attribute::NoCapture);
-    AttrList = AttrList.addFnAttribute(M.getContext(), Attribute::NoUnwind);
+    auto &C = M.getContext();
+
+    AttributeList AttrList;
+    AttrList = AttrList.addFnAttribute(C, Attribute::NoUnwind);
+    AttrList = AttrList.addParamAttribute(
+        C, 0, Attribute::getWithCaptureInfo(C, llvm::CaptureInfo::none()));
```

Comment:
Is the 0 vs 1 here correct? (I haven't looked at the API change)

Reply:
Yes, `addParamAttribute` forwards to `addAttributeAtIndex` with index + 1.

Reply:
The gist of the API change is that we can no longer express `captures(none)` with just an attr kind.